examples/40_advanced/example_custom_ensemble.py

```diff
@@ -0,0 +1,781 @@
+# -*- encoding: utf-8 -*-
+"""
+===============
+Custom Ensemble
+===============
+
+This example shows how to provide a custom ensemble to Auto-sklearn.
```

Comment:
Can we add a little bit more of how it's done, it's not really clear from first glance what all the other stuff about `WeightedMetricScorer` and such is needed before even talking instansiating the `MultiObjectiveEnsembleWrapper`.

Reply:
I wouldn't add this to Auto-sklearn so I'd leave documenting this to later.